RELEASE_NOTES.md

```diff
@@ -14,6 +14,10 @@ time.
 * You need to make sure to use [timezone-aware] `datetime` objects when using
   the timestamp returned by [`Timer`], Otherwise you will get an exception.
 
+* Channels methods `get_receiver()` and `get_sender()` have been renamed to
+  `new_receiver()` and `new_sender()` respectively. This is to make it more
+  clear that new objects are being created.
+
```

Comment:
Should this be listed under `Breaking changes`?

Reply:
Added a ` (breaking changes)` to `Upgrading` to make it more clear that this section list breaking changes. We can change the template if `Upgrading` is not clear enough. I used this to make it clear for people writing notes that it is important to say how to upgrade from breaking changes, not just listing them.